podcasts.md

```diff
@@ -0,0 +1,21 @@
+# Podcasts
+
+Here is a list of recommended podcasts:
```

Comment:
Maybe we should use the word "suggested"? "Recommended" comes a little too close to condoning, for podcasts that I'm sure most of the team haven't listened to, and definitely not exhaustively.